### DISCUSSION TITLE
Shapes & Dividers / Animation Not available

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1175] [Q&A] [answered]
I just upgraded a RC1 site to a RC2 site.  I have done nothing other than the upgrade.  I noticed on your Test Drive site that  "Shapes & Dividers / Animation"  are available selections on the Styling tab for the test drive site,  but on my site they are not available. Have I missed a setting somewhere??

Also just noticed that the BLOG BLOG WIDGET and FORM BUILDER Blocks are not in the Blocks tab

**Comment by meetmandeep** [comment, marked as answer]
@scottw12 Our online test site has our commercial Premium Theme applied which has many new features including new blocks (Tabs, Accordion, Counter), Styling (Animations, Shapes & Dividers), and Mega Menu. 

Similarly, the Apps (Blog, Blog Widget, and Form Builder) are commercial apps that are designed for Vanjaro. For example, the Blog app is our adaptation of Live Blog but it leverages the Vanjaro Blocks for blog publishing rather than using CK Editor. 

For our online testing, these features were supposed to be hidden; apparently, that's not the case :) I'll notify the team to hide these features and we'll release more information when it's available. 

So far, the following commercial extensions are in pipeline. They will be available in Vanjaro App Store after Vanjaro officially releases:

- Premium Theme
- Blog App
- Form Builder App
- Email Marketing App
- eCommerce App
- Social Authentication (Facebook / Google / LinkedIn)

**Comment by meetmandeep** [comment]
All Mandeeps.com modules have been updated and are compatible with Vanjaro. You may install and use them on Vanjaro. 

Vanjaro is a separate company and distinct from Mandeeps.com. It will have its own apps that are made only for Vanjaro. These will be sold separately. These will be licensed on a subscription model and will be available via Vanjaro Store. 

These are excellent questions to bring to our upcoming webinar on May 19th, 2021. We would love to talk to our customers and other stakeholders.
